recipes/pytest-timeout/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pytest-timeout" %}
+{% set version = "1.0.0" %}
+{% set hash_type = "sha256" %}
+{% set hash = "1465096be73e16df1e15d1b1453692428a7e15b997d756bc565aee0d12798ce1" %}
+{% set ext = "tar.gz" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ ext }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ ext }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools    
+  run:
+    - python
+    - pytest >=2.8.0
+
+test:
+  requires:
+    - pytest
+  imports:
+    - pytest_timeout
+  commands:
+    - py.test --help
+
+about:
+  home: https://bitbucket.org/pytest-dev/pytest-timeout
+  license: MIT
+  summary: This is a plugin which will terminate tests after a certain timeout.
+
+extra:
+  recipe-maintainers:
+    - nicoddemus
+    - flub
```

Comment:
@kalefranz since you submitted #819 do you want to be added here?

@nicoddemus I see that you are not running the full test suite as #819. That is probably OK and I believe the failure in AppVeyor there is unrelated to the package usability. Yet, can you comment on why you chose this path before we merge this?